metrics/alertmanager/pd.rules.yml

```diff
@@ -195,3 +195,27 @@ groups:
       description: 'cluster: ENV_LABELS_ENV, instance: {{ $labels.instance }}, values:{{ $value }}'
       value: '{{ $value }}'
       summary: PD_cluster_slow_tikv_nums
+
+  - alert: PD_cpu_quota
+    expr:  irate(process_cpu_seconds_total{job="pd"}[30s]) / pd_service_maxprocs > 0.8
```

Comment:
Do we need to specify the job of `pd_service_maxprocs` here?

Reply:
> Do we need to specify the job of `pd_service_maxprocs` here?

Thanks for the detailed review. It worked well as per tested. And as you can see, there are also other metrics in the same file w/o `job="pd"` specified. The metrics like `process_cpu_seconds_total` and `process_resident_memory_bytes` need to specify `job="pd"` to distinguish from other components because they are basic metrics provided by node exporter rather than the customized ones the components provide by themselves.

Reply:
Ok, I'm going to merge it. If needed, I might add it later.

Reply:
> Ok, I'm going to merge it. If needed, I might add it later.

Sure. Thank you for all your time on this PR.